vulkaninfo/CMakeLists.txt

```diff
@@ -17,17 +17,36 @@
 
 # CMakeLists.txt file for building Vulkaninfo
 
+find_package(PythonInterp 3 QUIET)
+set (PYTHON_CMD ${PYTHON_EXECUTABLE})
+
+set(VKINFO_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(KVULKANTOOLS_SCRIPTS_DIR ${CMAKE_SOURCE_DIR}/scripts)
+
+if(PYTHONINTERP_FOUND)
+    add_custom_target(generate_vulkaninfo_hpp
```

Comment:
Ideally this would be integrated with the existing generate_source.py and VulkanTools_generated_source target

_HOWEVER_

generate_source.py isn't smart enough today to handle multiple output directories, my bad.  Long term I would like to unify generate_source.py across the repos and drive it from a config file like we do for update_deps.py. That would be a good time to clean this up too.